applications/serial_lte_modem/doc/MQTT_AT_commands.rst

```diff
@@ -143,20 +148,46 @@ Response syntax
   It indicates the MQTT broker port.
 * The ``<sec_tag>`` value is an integer.
   It indicates the credential of the security tag used for establishing a secure connection.
+* The ``<keepalive>`` value is an integer.
+  It indicates the MQTT KeepAlive value (in seconds) used in current MQTT connection.
 
 Examples
 ~~~~~~~~
 
 ::
 
    AT#XMQTTCON?
-   #XMQTTCON: "MyMQTT-Client-ID","","","mqtt.server.com",1883
+   #XMQTTCON: 1,"MyMQTT-Client-ID","","","mqtt.server.com",1883,60
    OK
 
 Test command
 ------------
 
-The test command is not supported.
+The test command tests the existence of the command and provides information about the type of its subparameters.
+
+Syntax
+~~~~~~
+
+::
+
+   #XMQTTCON=?
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XMQTTCON: (0,1,2),<client_id>,<username>,<password>,<url>,<port>,<sec_tag>
```

Comment:
These should be explained. If they're the same as for the Set command, you could use an `.. include::` like was recently added for the GNSS AT commands.

Reply:
The response should be
#XMQTTCON: (list of op),<client_id>,<username>,<password>,<url>,<port>,<sec_tag>